gke-deploy/core/resource/templates.go

```diff
@@ -16,6 +16,20 @@ const (
 	namespaceTemplate = `apiVersion: v1
 kind: Namespace
 metadata:
-  name:
+  name: %s
+`
+
+	serviceTemplate = `apiVersion: v1
+kind: Service
+metadata:
+  name: %s
```

Comment:
Do we need to specify the namespace of this service here? This should match the target cluster namespace

Reply:
Spoke in person